import { CustomException } from "./setup/custom.exception";

export class NotFoundException extends CustomException {

    constructor(message = 'NÃ£o encontrado', statusCode = 404) {
        super(statusCode, message);
    }
}